shared-libs/couch-request/src/couch-request.js

```diff
@@ -0,0 +1,84 @@
+const request = require('request-promise-native');
+
+const isPlainObject = (value) => {
+  if (typeof value !== 'object' || value == null) {
+    return false;
+  }
+  const prototype = Object.getPrototypeOf(value);
+  return (prototype === null || prototype === Object.prototype || Object.getPrototypeOf(prototype) === null)
+    && !(Symbol.toStringTag in value) && !(Symbol.iterator in value);
+};
```

Comment:
Surely there's a lodash function for this?

Reply:
Last I checked the lodash implementation drags in a bunch of dependencies. Importing lodash for one function seems heavyweight.